ledger-core/virtual/execute/execute_test.go

```diff
@@ -488,3 +492,73 @@ func TestSMExecute_VCallResultPassedToSMObject(t *testing.T) {
 
 	mc.Finish()
 }
+
+func TestSendVStateReportWithMissingState_IfConstructorWasInterruptedBeforeRunnerCall(t *testing.T) {
+	t.Log("C5084")
+	defer testutils.LeakTester(t,
+		goleak.IgnoreTopFunction("github.com/insolar/assured-ledger/ledger-core/runner.(*worker).Run.func1"),
```

Comment:
this is probably the reason for PLAT-467

@bigbes we will need to introduce proper stopping of runner workers

Reply:
ehm, Abort should stop that goroutine. I've checked it in runner test.
It doesn't?

Reply:
Ah, you mean mocked runner. Sure.

Reply:
That's part of greater task - properly stop all adapters